docs/specs/authorization.md

```diff
@@ -9,13 +9,9 @@ Each authorization plugin must have a file called `index.js`.  Its purpose is to
 
 ### Function: getAuthenticationSpecification
 
-Authorization plugins are required to return a function called "getAuthenticationSpecification".  Its purpose is to configure a function that is used to describe the authentication specification. It should have the following signature:
+Authorization plugins are required to return a function called "authenticationSpecification".  Its purpose is delivery of an authorization/authentication options object that can be attached to the Model prototype for use in output-services (e.g., koop-output-geoservices). It should have the following signature:
```

Comment:
I thought this was made optional?

Reply:
No, I wasn't able to make it optional.  But I did simplify it.  Here's my comment from the other day; sorry I thought you saw it https://github.com/koopjs/koopjs.github.io/pull/29#discussion_r192770504